scalableminds/webknossos
# fix disappearing sidebar button

PR fix a long-standing issues where the little button to toggle the right-hand side panel would disappear when using the browser's builtin zoom. The issue is related to a float vs int comparison. 

### URL of deployed dev instance (used for testing):
- https://sidebarbutton.webknossos.xyz

### Steps to test:
- Load any dataset
- Use the browser zoom
- Before: the little button disappears in some zoom scales and re-appears in others
- Now: little button always here

### Issues:
- fixes #5564

------
(Please delete unneeded items, merge only when none are left open)
